packages/mdc-textfield/foundation.js

```diff
@@ -78,6 +79,8 @@ class MDCTextFieldFoundation extends MDCFoundation {
     this.receivedUserInput_ = false;
     /** @private {boolean} */
     this.useCustomValidityChecking_ = false;
+    /** @private {boolean} */
```

Comment:
You won't need `this.labelFloatIsActive_` after you change this to a const with the `updateLabelFloat_` method